### PR TITLE
[asset] add assets pages

### PR DESCRIPTION
**Pull request by nandofw**
**Assets**
added assets page show assets balance
added create assets page

**Others**
removed duplicated text on protx help
fixed background color on peers details
hide vertical header in smartnode list
Toolbar orientation changed to fit more pages

create asset page
Captura de tela de 2023-08-10 23-06-04

assets balance page
Captura de tela de 2023-08-10 23-17-04

### CODE REVIEW

File: src/evo/providertx.cpp [hunk 1]
```diff
@@ -179,8 +183,14 @@ bool CheckNewAssetTx(const CTransaction &tx, const CBlockIndex *pindexPrev, CVal
     }
 
     int hashlen = assetTx.referenceHash.length();
-    if ((hashlen != 0 && hashlen != 46) || (hashlen == 46 && assetTx.referenceHash.substr(0, 2) != "Qm")) {           
+    //ipfs alway start with 'Qm' and length 46
+    if ((hashlen == 46 && assetTx.referenceHash.substr(0, 2) != "Qm")) {
```
Comment (npq7721) [inline src/evo/providertx.cpp]: i think we discuss to support more then just ipfs hash so start wtih qm should not be required

File: src/qt/assetcontroltreewidget.h [hunk 2]
```diff
@@ -1,9 +1,9 @@
-// Copyright (c) 2011-2014 The BitAsset Core developers
+// Copyright (c) 2011-2014 The Bitcoin Core developers
```
Comment (npq7721) [inline src/qt/assetcontroltreewidget.h:1]: we need to add raptoreum copyright here

File: src/qt/assetcontroltreewidget.cpp [hunk 3]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2011-2015 The BitAsset Core developers
+// Copyright (c) 2011-2015 The Bitcoin Core developers
```
Comment (npq7721) [inline src/qt/assetcontroltreewidget.cpp:1]: we need to add raptoreum copyright here

File: src/qt/assetcontroldialog.h [hunk 4]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2011-2015 The BitAsset Core developers
+// Copyright (c) 2011-2015 The Bitcoin Core developers
```
Comment (npq7721) [inline src/qt/assetcontroldialog.h:1]: we need to add raptoreum copyright here

File: src/qt/assetcontroldialog.cpp [hunk 5]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2011-2015 The BitAsset Core developers
+// Copyright (c) 2011-2015 The Bitcoin Core developers
```
Comment (npq7721) [inline src/qt/assetcontroldialog.cpp:1]: we need to add raptoreum copyright here

**Review by npq7721 — CHANGES_REQUESTED** [review]

File: src/qt/assetcontroldialog.cpp (hunk 5 above)
Comment (nandofw) [reply src/qt/assetcontroldialog.cpp:1]: this file already has the Raptoreum copyright headers

File: src/qt/assetcontroltreewidget.cpp (hunk 3 above)
Comment (nandofw) [reply src/qt/assetcontroltreewidget.cpp:1]: added

File: src/qt/assetcontroldialog.h (hunk 4 above)
Comment (nandofw) [reply src/qt/assetcontroldialog.h:1]: added

File: src/qt/assetcontroltreewidget.h (hunk 2 above)
Comment (nandofw) [reply src/qt/assetcontroltreewidget.h:1]: added

File: src/evo/providertx.cpp (hunk 1 above)
Comment (nandofw) [reply src/evo/providertx.cpp]: fixed

**Review by npq7721 — APPROVED** [review]

**Pull request merged**